tests/python/ExponentTransformTest.py

```diff
@@ -0,0 +1,153 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+
+import unittest
+
+import PyOpenColorIO as OCIO
+
+
+class ExponentTransformTest(unittest.TestCase):
+    TEST_ID = 'sample exponent'
+    TEST_VALUES = [1, 2, 3, 4]
+    TEST_NEGATIVE_STYLE = OCIO.NEGATIVE_MIRROR
+    TEST_DIRECTION = OCIO.TRANSFORM_DIR_INVERSE
+
+    def setUp(self):
+        self.exp_tr = OCIO.ExponentTransform()
+
+    def tearDown(self):
+        self.exp_tr = None
+
+    def test_direction(self):
```

Comment:
To avoid repeating this method in every transform test, you may want to either:
- Create a Transform `TestCase` base class with this and any other common tests, which child transform classes can then just inherit without explicit implementation.
- Create a utility function to test direction and call it from each transform class.

Reply:
Note: This could be done in a later PR.

Reply:
I was under the impression that we want to avoid having the individual tests depend on something else, as such, I did not consider creating a base class or utility function. Given your suggestion, I am happy to look into one of those options to reduce code duplications. Thank you for your suggestion.

Reply:
I think this case justifies it to avoid having this same method across all the transform tests.

Reply:
Also thinking now about whether to also include validate() of direction in this utility/base class for testing transform direction? The assumption is that transform direction must be forward or inverse and not unknown. Therefore, I think it makes sense to include a validate test every time transform direction is tested. So it would make sense to bundle those two together in a utility/base class, right?

Reply:
That sounds fine to me.